backend/.env.txt

```diff
@@ -0,0 +1,2 @@
+PORT=8080
+DATABASE_URI=mongodb+srv://admin:Password123@workoutapp.7s766.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
```

Comment:
This file should not be pushed up. Also, it shouldn't be .env.txt just env

Reply:
Deleted file from PR